internal/storage/bucket_handle.go

```diff
@@ -205,6 +205,9 @@ func (bh *bucketHandle) CreateObject(ctx context.Context, req *gcs.CreateObjectR
 	// Chuck size for resumable upload is default i.e. 16MB.
 	wc := obj.NewWriter(ctx)
 	wc = storageutil.SetAttrsInWriter(wc, req)
+	wc.ProgressFunc = func(bytesUploadedSoFar int64) {
+		logger.Tracef("gcs: Req %#16x: -- CreateObject(%q): %16v bytes uploaded so far", ctx.Value(gcs.ReqIdField), req.Name, bytesUploadedSoFar)
```

Comment:
why are you formatting with %16 ?

Reply:
This is to match the formatting of the original debug logs for CreateObject in DebugBucket (https://github.com/GoogleCloudPlatform/gcsfuse/blob/852953d45fa1f39372e95dfa30aad4885c3dd2c0/internal/storage/debug_bucket.go#L57) .

Reply:
may be there 16 is used to print only 16 digits from uint64. i dont think we should restrict it for bytes,

Reply:
Oh you meant the later %16, I mistook it to think you meant the first one. 

Yeah, I somehow had it in my head that %16 is sufficient to cover all uint64s, but yeah it isn't. I'll change it to %20v to keep it aligned as well as sufficient.

Reply:
why to even restrict to 20? just print the entire value.

Reply:
> why to even restrict to 20? 

It is not to restrict, it is to align the print of the value across consecutive logs.

Aligned
```
gcs: Req             0x2f: -> StatObject(\"sample_240MB.dat\") (24.045451ms): OK"
gcs: Req             0x30: <- CreateObject(\"sample_240MB.dat\")"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):             16777216 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):             33554432 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):             50331648 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):             67108864 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):             83886080 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            100663296 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            117440512 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            134217728 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            150994944 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            167772160 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            184549376 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            201326592 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            218103808 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            234881024 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"):            251658240 bytes uploaded so far"
gcs: Req             0x30: -> CreateObject(\"sample_240MB.dat\") (1.621559367s): OK"
```

vs un-aligned
```
gcs: Req             0x30: <- CreateObject(\"sample_240MB.dat\")"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 16777216 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 33554432 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 50331648 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 67108864 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 83886080 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 100663296 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 117440512 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 134217728 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 150994944 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 167772160 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 184549376 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 201326592 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 218103808 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 234881024 bytes uploaded so far"
gcs: Req             0x30: -- CreateObject(\"sample_240MB.dat\"): 251658240 bytes uploaded so far"
gcs: Req             0x30: -> CreateObject(\"sample_240MB.dat\") (1.664684055s): OK"
```

Reply:
what happens if the value (bytesuploaded so far ) is greater than 20 digits?

Reply:
bytesuploaded is int64 which fits within 20 digits (including the negative sign, not that it'll ever be relevant in this case) including its min value, -9,223,372,036,854,775,808 .